src/app/app.component.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Adapted from original source:
+ *  <https://github.com/SimonBiggs/scriptedforms/blob/master/scriptedforms/src/app/app.component.ts>
```

Comment:
The license on that repo is scary. I think this PR is OK, but wanted to flag that we should avoid any dependency on it.

Reply:
Yeah, definitely. I've only been copying the code with the Apache license.